enhancements/oc/mirroring-manifests.md

```diff
@@ -105,78 +98,104 @@ mirror
+	```
+7. Use oc to login to a given cluster to be upgraded.
+8. Apply the the mirrored configmap, noted above, to the connected cluster for use by CVO for upgrade image verification:
+	```console
+	$ oc apply -f <removable-media-path>/mirror/manifests/sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+	```
+9. Perform the upgrade:
+	```console
+	$ oc adm upgrade --to-image REGISTRY/REPOSITORY/release@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+	```
 
 ### User Stories
 
 #### Release-signature ConfigMaps
 
-FIXME
+Currently a cluster upgrade can be accomplished on a cluster that does not have an active connection to the internet.
+However manual steps are required to create a ConfigMap containing the signature data required for update image verification.
+This enhancement will automatically create the ConfigMap from the image manifest files.
 
 #### In-cluster Cincinnati
 
 FIXME
 
 ### Risks and Mitigations
 
-FIXME
+One risk considered is that a user familiar with the current behavior of the `oc adm release mirror ...` command may inadvertently apply manifests to thier cluster.
+This has been addressed by requiring the explicit `--apply-manifests` argument to apply manifests to a cluster.
 
 ## Design Details
 
 ### Test Plan
 
-FIXME
+The current e2e test, that makes sure the command always exits successfully and that certain apsects of the content are always present, will be modified to validate the new functioanlity as well.
 
 ### Graduation Criteria
 
-FIXME
-
 ### Upgrade / Downgrade Strategy
 
-FIXME
+The image is included in the payload, but has no content running in a cluster to upgrade.
 
 ### Version Skew Strategy
 
-The proposal for [applying directly to the target cluster](#applying-directly-to-the-target-cluster) adds a new dependency on the cluster to which `oc` connects for Kubernetes activity.
-The explicit `--apply-manifests` argument requires users who have been using the previous `oc adm release mirror` implementation (which did not push manifests) to explicitly opt in to the new functionality, so there is no change that they accidentally pushing manifests into the wrong cluster.
-FIXME: check to see how this works if there are conflicts but neither `--force-conflicts` not `--overwrite` was set.
+The proposal for [applying manifests directly to the target cluster](#applying-manifests-directly-to-the-target-cluster) adds a new dependency on the cluster to which `oc` connects for Kubernetes activity.
+The explicit `--apply-manifests` argument requires users who have been using the previous `oc adm release mirror` implementation (which did not push manifests) to explicitly opt in to the new functionality, so there is no chance that they accidentally push manifests into the wrong cluster.
 
 ## Implementation History
 
-FIXME
-
 ## Drawbacks
 
-FIXME
-
 ## Alternatives
 
-FIXME
```

Comment:
Talk about serving signatures from Cincinnati (although I still think we need something attached to `oc adm release mirror ...` to gather external data for shipping into the cluster, even if that external data was a Cincinnati graph-with-signatures snapshot).